src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -57,6 +65,12 @@
         addMapping(MAPPING_TOWER_GUARD, JobGuard.class);
     }
 
+    //fix for the annotation
+    static
+    {
+        updateMapping();
+    }
+
     private final CitizenData citizen;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)